core/platform/azure_speech_synthesis/azure_speech_synthesis_services.py

```diff
@@ -50,14 +51,10 @@
 """
 
 # A string template block representing the math content within the SSML.
-MATH_TEMPLATE_SSML_BLOCK = """
-    <say-as interpret-as="math">
-        %s
-    </say-as>
-"""
+MATH_TEMPLATE_SSML_BLOCK = """<say-as interpret-as="math">%s</say-as>"""
 
 # A string template block representing the non math content within the SSML.
-NON_MATH_TEMPLATE_SSML_BLOCK = """
+SSML_TEMPLATE_BLOCK = """
```

Comment:
Just curious, why are we updating this? Based on the comment above, the previous name seems OK.

Reply:
Updated the code comment and variable name. Thank you!